gethwrappers/payments/go_generate.go

```diff
@@ -0,0 +1,7 @@
+// Package gethwrappers provides tools for wrapping solidity contracts with
```

Comment:
Does it make sense to create a frozen geth wrapper for the current version? E.g. in CCIP style:

```
gethwrappers/payments/generated/payment_token_on_ramp_v1_0_0/payment_token_on_ramp.go
```

Or in functions style:
https://github.com/smartcontractkit/chainlink-evm/tree/develop/contracts/src/v0.8/functions/v1_3_0